### ISSUE TITLE
Added GitHub to my account, but it's organisations are not displayed in the subscriptions menu

### ISSUE THREAD

**Issue by slavafomin** [Shippable/support#3532]
Hello!

I've just connected my GitHub account to an existing Shippable account (previously logged in through BitBucket).

However, when I open subscriptions menu there is no organisations from my GitHub account there (I'm sure I've allowed access to them when connecting).

**Comment by slavafomin** [comment]
UPDATE: I've opened account configuration and found `Sync` button there. After pressing it, organisations appeared in the menu. I think it should be done automatically when new connection was added.

Also, after setting up a connection with GitHub, I've entered the account settings and found that it wasn't completed or something. There was a button near my GitHub account. I've pressed it and gave yet another permission.

Long story short: it was a very counter-intuitive experience connecting GitHub account. I would suggest to review it and make it more straightforward and automatic.

Thank you!

**Comment by ambarish2012** [comment]
@slavafomin thanks for the feedback.
